packages/components/src/color-indicator/style.scss

```diff
@@ -1,11 +1,8 @@
 .component-color-indicator {
-	width: 25px;
-	height: 16px;
-	margin-left: 0.8rem;
-	border: 1px solid #dadada;
+	width: $grid-unit-50 * 0.5;
+	height: $grid-unit-50 * 0.5;
+	border-radius: 50%;
+	border: $border-width solid $gray-300;
 	display: inline-block;
-
-	& + & {
-		margin-left: 0.5rem;
-	}
```

Comment:
This change could cause some layout glitches in other parts of Gutenberg

Reply:
I haven't found any places where this rule would apply. As is, I removed it because it appears to be dead CSS.

Reply:
I'm not sure it's dead CSS — this rule would potentially affect those situations where 2 or more `ColorIndicator`s are rendered next to each other (at least in DOM order).

I scanned the codebase and found a few places that we should check to make sure we didn't introduce a layout regressions:

- [`PanelColorGradientSettings`](https://github.com/WordPress/gutenberg/blob/26556937b70bbf4aaef8c112321703b61a740284/packages/block-editor/src/components/colors-gradients/panel-color-gradient-settings.js#L75)
- Global Styles' [`Palette`](https://github.com/WordPress/gutenberg/blob/233e5b2fa55363a0343f865b697c7923d49eb017/packages/edit-site/src/components/global-styles/palette.js#L66-L69) (although this may be already tackled by the changes to `packages/edit-site/src/components/global-styles/style.scss` in this same PR)
- Global Styles' [`Preview`](https://github.com/WordPress/gutenberg/blob/251f00f45688fdb25e883e861b5ac788d58ec84b/packages/edit-site/src/components/global-styles/preview.js#L35-L36) (although this may be OK because of the `VStack` element wrapping around them)